pkg/storage/db.go

```diff
@@ -1060,3 +1083,83 @@ order by ae.ts asc
 	}
 	return nil
 }
+
+// StoreAccountOwner is an implementation of AccountOwnerStorer interface that saves account ID, its owner and champions of the account to a database
+func (db *DB) StoreAccountOwner(ctx context.Context, accountOwner domain.AccountOwner) error {
+
+	tx, err := db.sqldb.Begin()
+	if err != nil {
+		return err
+	}
+
+	err = db.storeAccountOwner(ctx, accountOwner, tx)
+
+	if err != nil {
+		if rollbackErr := tx.Rollback(); rollbackErr != nil {
+			return errors.Wrap(rollbackErr, err.Error()) // so we don't lose the original error
+		}
+		return err
+	}
+	if err = tx.Commit(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (db *DB) storeAccountOwner(ctx context.Context, accountOwner domain.AccountOwner, tx *sql.Tx) error {
+
+	sqlStatement := fmt.Sprintf(`REPLACE INTO aws_account (account) VALUES($1)`)
```

Comment:
I've never seen REPLACE INTO before, is this like an upsert? It creates if it doesn't exist and updates if it does?

Reply:
It works like upsert, except it will delete the values of old records and insert new values, if primary key or unique key in old records are the same as new records.

Reply:
Actually PostgresSQL does not have REPLACE INTO, and MySQL has it. Upsert is a way to do the job. I will make changes to the query. Thank you for your comment.